src/network/netdev/bond.c

```diff
@@ -371,6 +390,116 @@ int config_parse_arp_ip_target_address(const char *unit,
         return 0;
 }
 
+int config_parse_ad_actor_sys_prio(const char *unit,
+                                   const char *filename,
+                                   unsigned line,
+                                   const char *section,
+                                   unsigned section_line,
+                                   const char *lvalue,
+                                   int ltype,
+                                   const char *rvalue,
+                                   void *data,
+                                   void *userdata) {
+        Bond *b = userdata;
+        uint16_t v;
+        int r;
+
+        assert(filename);
+        assert(lvalue);
+        assert(rvalue);
+        assert(data);
+
+        r = safe_atou16(rvalue, &v);
+        if (r < 0) {
+                log_syntax(unit, LOG_ERR, filename, line, r, "Failed to parse actor system priority '%s', ignoring: %m", rvalue);
+                return 0;
+        }
+
+        if (v == 0) {
+                log_syntax(unit, LOG_ERR, filename, line, r, "Failed to parse actor system priority '%s'. Range is [1,65535], ignoring: %m", rvalue);
+                return 0;
+        }
+
+        b->ad_user_port_key = v;
```

Comment:
It seems that `ad_actor_sys_prio` should be here instead of `ad_user_port_key`. It's most likely why the test is failing.

Reply:
oh dear... sorry for that.